theme/themes/pastanaga/extras/contents.less

```diff
@@ -78,8 +78,14 @@
     top: 3px;
   }
 
+  .ui.upward.dropdown > .menu {
+    top: 100%;
+    bottom: inherit;
+  }
+
   .contents-table-wrapper {
     width: 100%;
+    min-height: 13.8888rem;
```

Comment:
@SaraBianchi why not simply use 14rem?
Looking here https://volto.kitconcept.com/contents
**13.8888rem** amounts  to a subpixel value **194.443px**
while **14rem** equals to **196px**

Reply:
I don't remember, maybe I made my calculations wrong. No problem, we can change it